backend/src/main/java/com/huellapositiva/domain/model/entities/Reviser.java

```diff
@@ -0,0 +1,19 @@
+package com.huellapositiva.domain.model.entities;
+
+import com.huellapositiva.domain.model.valueobjects.EmailAddress;
+import com.huellapositiva.domain.model.valueobjects.Id;
+
+public class Reviser extends User {
+
+    public Reviser(EmailAddress emailAddress, Id id) {
```

Comment:
Hi Manu, what do you mean? 🎵

Reply:
From a code perspective, the code looks good as it is. I have no argument against it. However Lombok can support with certain common operations like the constructors.
If you want to play with it, I'd look at @NoArgsConstructor, @AllArgsConstructor annotations and see if you can support on them to make the code look nicer.

Reply:
Hi Manu! Thank you for your comment. 
We had a look at it, and it seems we can't use those annotations, because those constructors extend those in USER.java. Or do you think we can still use them?? Thanks for your help :)

Reply:
I think you are right. Looking at the documentation in: https://projectlombok.org/features/constructor One can see the different configurations that can be added to the mentioned annotations.
Some lombok annotations allow the flag `callSuper = true` However this feature seems not available to build the Object :-)
It is good as it is. Today we've lean something new.